dts/arm/st/f0/stm32f030.dtsi

```diff
@@ -7,6 +7,9 @@
 #include <st/f0/stm32f0.dtsi>
 
 / {
+	soc {
+		compatible = "st,stm32f030", "st,stm32f0", "simple-bus";
+	};
```

Comment:
Would you mind breaking a line here ?

Reply:
Hi, just to make sure, since I will have to apply this change to all of the modified files - is this what you have in mind?
```
	soc {
		compatible = "st,stm32f030", "st,stm32f0",
			     "simple-bus";
	};
```

Reply:
Got it, thanks for elaborating. Should be fixed now.